packages/babel-plugin/__tests__/transform-stylex-defineVars-test.js

```diff
@@ -584,11 +587,11 @@ describe('@stylexjs/babel-plugin', () => {
           "import _inject from "@stylexjs/stylex/lib/stylex-inject";
           var _inject2 = _inject;
           import * as stylex from '@stylexjs/stylex';
-          _inject2(":root, .xm1nzai{--xjrzwe6:red;--x1ybzi8d:green;--x1o4mfbm:blue;}", 0);
+          _inject2(":root, .xm1nzai{--color-xjrzwe6:red;--nextColor-x1ybzi8d:green;--otherColor-x1o4mfbm:blue;}", 0);
           export const vars = {
-            color: "var(--xjrzwe6)",
-            nextColor: "var(--x1ybzi8d)",
-            otherColor: "var(--x1o4mfbm)",
+            color: "var(--color-xjrzwe6)",
+            nextColor: "var(--nextColor-x1ybzi8d)",
+            otherColor: "var(--otherColor-x1o4mfbm)",
```

Comment:
This snapshot updated because the test no longer sets `debug:false` in the default options, which means `dev:true` should also produce debug data.